pkgs/development/python-modules/shapely/1.8.nix

```diff
@@ -64,16 +64,21 @@ buildPythonPackage rec {
     rm -r shapely # prevent import of local shapely
   '';
 
-  disabledTests = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
-    # FIXME(lf-): these logging tests are broken, which is definitely our
-    # fault. I've tried figuring out the cause and failed.
-    #
-    # It is apparently some sandbox or no-sandbox related thing on macOS only
-    # though.
-    "test_error_handler_exception"
-    "test_error_handler"
-    "test_info_handler"
-  ];
+  disabledTests =
+    [
+      # started to fail after geos update from 3.11.2 to 3.11.4
+      "test_parallel_offset_linestring"
```

Comment:
Is there an upstream issue, so that we can eventually remove this line?

Reply:
shapely 1.8 is old version. I guess it is out of support.